tests/test_event.py

```diff
@@ -45,6 +45,39 @@ def login_user(self):
         }
         return self.client().post('/api/auth/login', data=user_data)
 
+    def test_api_get_all_events(self):
+        """Test API can get all events without token passed (GET request)."""
+        self.register_user()
```

Comment:
You can create a method for registering, logging in user and getting the access token

Reply:
I already have them at the top of the file.

Reply:
for the whole process of Registering, Loging In and getting access token and avoid repeating all the process every time you want to get a access token

Reply:
okay I now understand your review